src/test/java/BaseTest.java

```diff
@@ -1,182 +1,125 @@
-        emailField.sendKeys(email);
+    public static void clickSubmitButton() {
+        WebElement submitButton = driver.findElement(By.cssSelector("button[type='submit"));
     }
 
-    public void clickSaveButton() {
-        WebElement saveButton = getDriver().findElement(By.cssSelector("button.btn-submit"));
-        saveButton.click();
+    public static void validateHomepage() {
+        WebElement avatarIcon = driver.findElement(By.cssSelector("img.avatar"));
+        Assert.assertTrue(avatarIcon.isDisplayed());
     }
 
-    public void provideProfileName(String randomName) {
-        WebElement profileName = getDriver().findElement(By.cssSelector("[name='name']"));
-        profileName.clear();
-        profileName.sendKeys(randomName);
+    public static void addPlaylist() {
+        WebElement addPlayList = driver.findElement(By.xpath("//*[@class='fa fa-plus-circle create']"));
     }
 
-    public void provideCurrentPassword(String password) {
-        WebElement currentPassword = getDriver().findElement(By.cssSelector("[name='current_password']"));
-        currentPassword.clear();
-        currentPassword.sendKeys(password);
+    public static void newPlaylist() {
+        WebElement newPlaylist = driver.findElement(By.xpath("//*[@data-testid='playlist-context-menu-create-simple']"));
     }
 
-    public String generateRandomName() {
-        return UUID.randomUUID().toString().replace("-", "");//
+    public static void namePlaylist() {
+        WebElement namePlaylist = driver.findElement(By.cssSelector("input[placeholder='↵ to save']"));
+        namePlaylist.clear();
+        namePlaylist.click();
+        namePlaylist.sendKeys("best songs");
+        namePlaylist.sendKeys(Keys.ENTER);
     }
 
-    public void clickAvatarIcon() {
-        WebElement avatarIcon = getDriver().findElement(By.cssSelector("img.avatar"));
-        avatarIcon.click();
-
+    public static void doubleClickToPlaylist() {
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".playlist:nth-child(3)")));
+        WebElement webElement = driver.findElement(By.cssSelector(".playlist:nth-child(3)"));
+        action.doubleClick(webElement).perform();
     }
 
-    @DataProvider(name = "incorrectLoginProviders")
-    public static Object[][] getDataFromDataproviders() {
-
-        return new Object[][]{
-                {"invalid@email.com", "invalidPass"},
-                {"demo@mail.com", "invalid"},
-                {"", ""}
-        };
+    public static void choosePlaylist() {
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".playlist:nth-child(3)"))).click();
     }
-}
+
+    public static void newPlaylistName() {
+        WebElement enterName = driver.findElement(By.cssSelector("input[name='name']"));
+        enterName.sendKeys(Keys.chord(Keys.COMMAND, "a", Keys.BACK_SPACE));
+        enterName.sendKeys("Wild dream");
+        enterName.sendKeys(Keys.ENTER);
+    }
```

Comment:
![A31-QA-Automation-Framework – BaseTest java  A31-QA-Automation-Framework test](https://user-images.githubusercontent.com/49252454/215113819-4cf51175-6d6a-425e-8540-6537bc0187cb.jpg)


Here you missed somthing